datafusion/expr/src/logical_plan/builder.rs

```diff
@@ -1138,6 +1139,31 @@ impl LogicalPlanBuilder {
         )?))
     }
 }
+
+/// Converts a `Arc<LogicalPlan>` into `LogicalPlanBuilder`
```

Comment:
FYI in general, the style of docs in Rust is to put them in triple quotes which will ensure they are actually run as part of the CI tests (`cargo test --doc` will run them)

So in this case like 

```
/// ```
/// fn employee_schema() -> Schema {
...
```

Any chance you are willing to make a follow on PR to change this to a docstring @AbrarNitk ?

Reply:
Yep, I will raise another PR.

Reply:
Hey @alamb, please check the PR: https://github.com/apache/datafusion/pull/10466